src/ServiceManagement/Profile/Commands.Profile/Models/PsAzureSubscription.cs

```diff
@@ -34,12 +34,10 @@ public PSAzureSubscription(AzureSubscription subscription, AzureSMProfile profil
             Accounts = profile.Accounts.Values.Where(a => a.HasSubscription(subscription.Id)).ToArray();
             IsDefault = subscription.IsPropertySet(AzureSubscription.Property.Default);
             IsCurrent = profile.Context != null && profile.Context.Subscription.Id == subscription.Id;
-            CurrentStorageAccountName = subscription.GetProperty(AzureSubscription.Property.StorageAccount);
+            CurrentStorageAccountName = GetAccountName(subscription.GetProperty(AzureSubscription.Property.StorageAccount));
```

Comment:
Should leave the property as is, we want to make sure current storage account is still transmitted with the full connection string